component/s3/file/inout.go

```diff
@@ -0,0 +1,46 @@
+package s3
```

Comment:
Naming the file inout.go is fine for now but I think we'll want to make things consistent eventually so it's easier to find code.

Reply:
changed to `types.go` to match scrape, so at least there is a hint of commonality ;)